src/Algebra/Lattice/Bundles.agda

```diff
@@ -0,0 +1,184 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions of algebraic structures like semilattices and lattices
+-- (packed in records together with sets, operations, etc.), defined via
+-- meet/join operations and their properties
+--
+-- For lattices defined via an order relation, see
+-- Relation.Binary.Lattice.
+------------------------------------------------------------------------
+
+-- The contents of this module should be accessed via `Algebra.Lattice`.
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Lattice.Bundles where
+
+open import Algebra.Core
+open import Algebra.Bundles
+open import Algebra.Structures
+open import Algebra.Lattice.Structures
+open import Relation.Binary
+open import Function.Base
+open import Level
+
+record Semilattice c ℓ : Set (suc (c ⊔ ℓ)) where
+  infixr 7 _∙_
+  infix  4 _≈_
+  field
+    Carrier       : Set c
+    _≈_           : Rel Carrier ℓ
+    _∙_           : Op₂ Carrier
+    isSemilattice : IsSemilattice _≈_ _∙_
+
+  open IsSemilattice isSemilattice public
+
+  band : Band c ℓ
+  band = record { isBand = isBand }
+
+  open Band band public using
```

Comment:
The updated style guide says `using` should go on the next line down (which I think you're doing elsewhere already)